src/utils.js

```diff
@@ -34,9 +34,13 @@ const parseInputTokens = (command, inputTokens) => {
       for (let j = 0; j < patternTokens.length; j++) { // for each token in pattern
         const patternToken = patternTokens[j];
         if (patternToken.isArg) {
-          args[patternToken.value] = inputTokens[j];
-        }
-        else {
+          if (/\$[A-Z][0-9A-Z]*/.test(inputTokens[j])) {
```

Comment:
any way to do this more cleanly, without passing constants as a function parameter to parseInputTokens?

Reply:
As I felt, if we are not sending the constant object to utils, we have to define the extraction logic in `index.js`. I'll check little bit more about that.

Reply:
@hliyan , shall I separate the concerns by declaring two util functions. One is to get the `constants` within the `line` and other is to replace the corresponding constants with values. Might be a overkill for such a small functionality. Any thoughts?